edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -0,0 +1,1349 @@
+
+Full Delegate is the highest level that most Delegates will attain in their career as a WCA staff member. However, there are additional managerial and supervision based roles for Delegates who wish to take on additional responsibilities within the WCA.
+
+### Regional Delegates
+
+Alongside the normal responsibilities of a Full Delegate, a Regional Delegate helps oversee and manage competitions in a broader region than their immediate local cubing community. They are responsible for maintaining quality standards in their local community and supporting their local Full Delegates, Junior Delegates, and Trainee Delegates in performing their respective roles. Regional Delegates are expected to act as a leader of Delegates within their region, to encourage continuous improvement of competition quality, and evaluate area’s needs for additional Delegates. A Regional Delegate’s responsibilities are largely the same as a Senior Delegate’s, but the smaller assigned region allows a Regional Delegate to pay closer attention to the needs of their region and their Delegates than a Senior Delegate may be able to.
+
+Regional Delegates are appointed by a Senior Delegate in conjunction with the WCA Board on an as-needed basis as determined by the Senior Delegate. A Regional Delegate’s appointment time is indefinite, although a Regional Delegate can step down from their position voluntarily or have their appointment removed by the WCA Board in consultation with the Senior Delegate if they do not fulfill their responsibilities.
+
+Not all regions have a Regional Delegate. In the case that no Regional Delegate exists, the Senior Delegate fulfills their responsibilities.
+
+### Senior Delegates
+
+A Senior Delegate is the highest level of Delegate within the WCA. They are responsible for overseeing the largest geographic divisions in the WCA as well as appointing Trainee Delegates and promoting Delegates to higher roles.
+
+A Senior Delegate is appointed by the WCA Board for a period of 24 months. After 24 months, the appointment automatically expires, unless extended by the WCA Board. Extensions can be of any length of time up to an additional 24 months. The maximum appointment length of a Senior Delegate is 48 months. Senior Delegates are able to re-apply for the position after the end of this period.
+
+Some regions may not have a Senior Delegate appointed. In the case that no Senior Delegate is appointed, the WCA Board fulfills their responsibilities.
+
+The regions with Senior Delegates are:
+
+- Africa
```

Comment:
Do you mean the regions with a Senior Delegate role? Because Africa currently has no Senior Delegate. I'd change the line above to say "The regions with a Senior Delegate role are:".

Reply:
Those regions still have a Senior Delegate role, it's just that the Senior Delegate is the Board.

Reply:
I didn't say they don't have the role, I said they have no Senior Delegate, hence my change suggestion

Reply:
I don't think it makes sense to say the Board is the Senior Delegate, even if it's technically correct

Reply:
It is technically correct though, so it makes sense to say